docs/modules/ROOT/pages/tmail-backend/features/openpaas-integration.adoc

```diff
@@ -0,0 +1,34 @@
+= OpenPaaS Integration
```

Comment:
How do this page interact with https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/pages/tmail-backend/features/contactAutocomplete.adoc ?

Maybe we can mention in contactAutocomplete.adoc `synchronization with external third-party contact app via RabbitMQ, and especially we provide an link[integration with OpenPaaS]`